src/app/clusters/time-synchronization-server/time-synchronization-server.h

```diff
@@ -124,9 +124,10 @@ class TimeSynchronizationServer : public FabricTable::Delegate
     void OnAttributeData(const ConcreteDataAttributePath & aPath, TLV::TLVReader * apData, const StatusIB & aStatus) override;
     void OnDone(ReadClient * apReadClient) override;
 
-    CHIP_ERROR AttemptToGetTimeFromTrustedNode();
 #endif
 
+    CHIP_ERROR AttemptToGetTimeFromTrustedNode();
```

Comment:
This seems to be a time sync specific function. Should we instead #ifdef out were it used insteand?

Reply:
That is another option

Reply:
This is fine - the internals of the function are ifdef'd.

Reply:
ie, 6 of one...